database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
+    table.string( 'user_name' ).notNullable()
+    table.string( 'organization_name' ).notNullable()
+    table.string( 'logo' )
```

Comment:
Is this the logo image data itself or the URL for the image?

If it's the URL, then it should be clear. Maybe column name `logo_url` or `logo_image_url`.